### PR TITLE
feat(protocol-designer): show deck state in batch edit hover and show "no shared settings"

### LINKED ISSUE #7455: PD multi-select: show deck map on hover
# Overview

When steps are hovered on the deck map should be shown

## Acceptance Criteria
- [ ] Null deck state is shown when user is in batch edit mode and no step is being hovered upon. Null deck state is the same size as the regular deck state. 

- [ ] When a step is being hovered upon then the deck displays at regular size below the "form" box

- Copy on top of form should read "Batch editing settings are only available for Transfers"

### PR DESCRIPTION
**Pull request by IanLondon**
# Overview

Closes #7455

# Changelog

- Also, blue border now goes all the way around the form

# Review requests

See #7455 for illustrations!

- [ ] Null deck state is shown when user is in batch edit mode and no step is being hovered upon. Null deck state is the same size as the regular deck state. 
- [ ] When a step is being hovered upon then the deck displays at regular size below the "form" box
- [ ] Copy on top of form should read "Batch editing settings are only available for Transfers"
- [ ] Reminder: we should probably add this to our E2E list?

# Risk assessment

Med, PD

### CODE REVIEW

**Review by Kadee80 — APPROVED** [review]
Looks good! Question:
What about when you have non editable steps selected? Should we show a message somewhere on hover still?

**Review by shlokamin — APPROVED** [review]
Thanks @IanLondon!

**Comment by IanLondon** [comment]
Converted to draft pending design review

**Review by Kadee80 — APPROVED** [review]
🧨 🧯 🔥 🚒

File: protocol-designer/src/components/StepSelectionBanner/StepSelectionBannerComponent.js [hunk 1]
```diff
@@ -4,26 +4,26 @@ import startCase from 'lodash/startCase'
...
   JUSTIFY_FLEX_START,
   JUSTIFY_SPACE_BETWEEN,
-  DIRECTION_COLUMN,
-  TEXT_TRANSFORM_UPPERCASE,
+  POSITION_STICKY,
   SIZE_2,
   SPACING_3,
-  POSITION_STICKY,
+  TEXT_TRANSFORM_UPPERCASE,
 } from '@opentrons/components'
```
Comment (shlokamin) [inline protocol-designer/src/components/StepSelectionBanner/StepSelectionBannerComponent.js:27]: `> Sort Lines Ascending` nice!

**Review by shlokamin — APPROVED** [review]
🚢

**Pull request merged**